app/routes/datasets/queries.py

```diff
@@ -336,8 +336,19 @@ async def query_dataset_list_post(
     request: QueryBatchRequestIn,
     # api_key: APIKey = Depends(get_api_key),
 ):
-    """Execute a READ-ONLY SQL query on the given dataset version (if
-    implemented)."""
+    """Execute a READ-ONLY SQL query on the specified raster-based dataset version
+    for a potentially large list of features. The features may be specified by an
+    inline GeoJson feature collection or the URI of a file in CSV or GeoJson format.
+    For CSV files, the geometry column should be named "WKT" and the geometry values
+    should be in WKB format.
+
+    The specified sql query will be run on each individual feature, and so may take a
+    while. Therefore, the results of this query include a job_id. The user should
+    then periodically query the specified job via the /job/{job_id} api. When the
+    "data.status" indicates "saved", then the results will be available at the specified
```

Comment:
I think we changed the statuses a bit but you weren't there. Instead of "saved", its now "success" if everything succeeded, or "partial_success" if there were some failed geometries

Reply:
Good catch, I think I was looking at an older result before that change was made, but should have remembered it was changed.  Updated the text to describe "success", "partial_success", "failure", and "error".  As you mentioned, we probably have a bunch of future changes to improve the output messages to indicate the actually kind of "error", such as timeout or other step function failure.